public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css"/>
```

Comment:
Why are you including the css using cdn AND importing it. 🤔

Reply:
Ok so i dont need both .. hehe , good to know ;)

Reply:
Removed.